curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/6601a8fb2e993b55912f9e9f.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 6601a8fb2e993b55912f9e9f
+title: Step 72
+challengeType: 20
+dashedName: step-72
+---
+
+# --description--
+
+The dot product between two 3D vectors \\( \vec{a} \\) and \\( \vec{b} \\) can be computed as it follows:
+
+\\[ \vec{a} \times \vec{b} = \begin{pmatrix} a_yb_z - a_zb_y \\\ a_zb_x - a_xb_z \\\ a_xb_y - a_yb_x \end{pmatrix}
+ \\]
+ 
+Where the resulting vector is represented as a column vector.
```

Comment:
Is it possible to show the final output in the console as a column vector?
Currently it is:
```py
v1 x v2 = (4.75, -3.5, 1.0) 
```

Reply:
I wrote it as a column vector in the formula just because it is easier to read.